python/lib/dependabot/python/update_checker/pip_compile_version_resolver.rb

```diff
@@ -216,10 +215,15 @@ def run_command(command, env: python_env)
           )
         end
 
+        def new_resolver_supported?
+          python_version.to_s.start_with?("3.6")
```

Comment:
Should this now be:

```suggestion
          !python_version.to_s.start_with?("3.6")
```

Or what about:

```suggestion
          python_version >= Python::Version.new("3.6")
```

Reply:
I think we'd have to do `python_version >= Python::Version.new("3.7")`.  This is the best option I think.